atchem.f90

```diff
@@ -218,8 +218,8 @@ end subroutine FCVFUN
   call set_model_parameters( getParametersFromFile( trim( param_dir ) //  "/model.parameters" ) )
   write (*,*)
 
-  ! Set the members dayOfYear, dayAsFractionOfYear, secondsInYear of
```

Comment:
Ok. So long as we end up with a value of `dec` and `zenith` that changes continuously, I'm happy.